pdns/recursordist/rec-main.cc

```diff
@@ -2750,8 +2750,7 @@ static void recursorThread()
       }
     }
 
-    unsigned int ringsize = ::arg().asNum("stats-ringbuffer-entries") / RecThreadInfo::numUDPWorkers();
-    if (ringsize != 0) {
+    if (unsigned int ringsize = ::arg().asNum("stats-ringbuffer-entries") / RecThreadInfo::numUDPWorkers(); ringsize != 0) {
```

Comment:
I don't like this syntax, but given this is a backport, I'm too late :)

Reply:
Oh, I like the combination of restricted score and condition very much.

Reply:
oh, I like that too. I just don't like how it looks. I do see how another layer of `{}` is not pretty either, but this form makes me think too hard ;)